public/pages/CreateTrigger/containers/DefineTrigger/AnomalyDetectorTrigger.js

```diff
@@ -0,0 +1,122 @@
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { get } from 'lodash';
+import { EuiSpacer, EuiLoadingChart } from '@elastic/eui';
+import { AnomalyDetectorData } from '../../../CreateMonitor/containers/AnomalyDetectors/AnomalyDetectorData';
+import TriggerExpressions from '../../components/TriggerExpressions';
+import { AnomaliesChart } from '../../../CreateMonitor/components/AnomalyDetectors/AnomaliesChart';
+import { FeatureChart } from '../../../CreateMonitor/components/AnomalyDetectors/FeatureChart/FeatureChart';
+import { EmptyFeaturesMessage } from '../../../CreateMonitor/components/AnomalyDetectors/EmptyFeaturesMessage/EmptyFeaturesMessage';
+
+class AnomalyDetectorTrigger extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { adValues } = this.props;
+    return (
+      <div style={{ padding: '0px 10px' }}>
+        <AnomalyDetectorData
+          detectorId={this.props.detectorId}
+          render={anomalyData => {
+            let featureData = [];
+            //Skip disabled features showing from Alerting.
+            featureData = get(anomalyData, 'detector.featureAttributes', [])
+              .filter(feature => feature.featureEnabled)
+              .map((feature, index) => ({
+                featureName: feature.featureName,
+                data: anomalyData.anomalyResult.featureData[feature.featureId] || [],
+              }));
+            const annotations = get(anomalyData, 'anomalyResult.anomalies', [])
+              .filter(anomaly => anomaly.anomalyGrade > 0)
+              .map(anomaly => ({
+                coordinates: {
+                  x0: anomaly.startTime,
+                  x1: anomaly.endTime,
+                },
+                details: `There is an anomaly with confidence ${anomaly.confidence}`,
+              }));
+            if (featureData.length > 0) {
+              return (
+                <React.Fragment>
+                  <TriggerExpressions
+                    thresholdValue={adValues.anomalyGradeThresholdValue}
+                    thresholdEnum={adValues.anomalyGradeThresholdEnum}
+                    keyFieldName="anomalyDetector.anomalyGradeThresholdEnum"
+                    valueFieldName="anomalyDetector.anomalyGradeThresholdValue"
+                    label="Anomaly grade condition"
+                  />
+                  <EuiSpacer size="xs" />
+                  <AnomaliesChart
+                    showTitle={false}
+                    showSettings={false}
+                    startDateTime={anomalyData.previewStartTime}
+                    endDateTime={anomalyData.previewEndTime}
+                    anomalies={anomalyData.anomalyResult.anomalies}
+                    isLoading={anomalyData.isLoading}
+                    displayGrade
+                    annotationData={[
+                      { dataValue: adValues.anomalyGradeThresholdValue, details: 'detail-0' },
```

Comment:
will we show 'detail-0' in annotation line?

Reply:
good catch, will remove this unused code.